server/container_create.go

```diff
@@ -491,111 +594,6 @@ func addSecretsBindMounts(ctx context.Context, mountLabel, ctrRunDir string, def
 	return mounts, nil
 }
 
-// CreateContainer creates a new container in specified PodSandbox
```

Comment:
I am very opposed to move this around -- this pollutes the git history, making it harder to read it. The justification for the move ("looks better when it's defined before being used") is not strong enough.

Reply:
it isn't really "looks better", I personally find it incredibly difficult to read when functions do not read in order of their calling. the entire server/ package is really hard to read. Though, I understand your point on the git history pollution. I can drop this for now, but do intend on finding a way to move around every function that is defined before its called, hopefully we can find a way that satisfies both of our expectations

Reply:
I agree that ideally and usually a function should be placed right above its use, and it should be done in any new code whenever possible, but I am opposed to moving the code around for the sole reason of placing it before it's used. I find myself frequently going through git history to find out when a particular line of code was introduced or changed, and there are not many tools to make it easier than repeating `git log file` and `git blame <revision> -- file` ad infinitum.

Not sure what kind of editor/IDE are you using, but I find vim + [vim-lsp](https://github.com/prabirshrestha/vim-lsp) + gopls incredibly helpful when reading the code. Jumping back and forth from function use to its implementation, or listing the function callers, etc, is a keystroke away, and it's relatively fast with gopls even with bigger codebases. I'd like to add that my past experience with vim-go and go-guru is not so good.

Even without the advanced tools referenced above, something as simple as `grep`, `git grep`, or a search function built into an editor is enough to find an identifier.